external/libp2p-service.js

```diff
@@ -258,13 +272,16 @@ class Libp2pService {
                 for await (const msg of source) {
                     bl.append(msg);
                 }
-                // console.log(`Receiving data using stream: ${result.toString()}`);
                 return bl;
             },
         )
 
-        // TODO: Remove - Backwards compatibility check with 1.30 and lower
-        if(response.toString() === constants.NETWORK_RESPONSES.ACK) {
+        if(timeoutController) {
+            timeoutController.signal.removeEventListener('abort');
+            timeoutController.clear();
+        }
+
+        if(response.toString() === '') {
```

Comment:
When will we get this response, when it times out? It would be useful to add comment above just so someone who first time reads the code can understand this check

Reply:
You're right. I will add this comment